core/server/worker/src/main/java/alluxio/worker/grpc/DoraWorkerClientServiceHandler.java

```diff
@@ -52,13 +57,27 @@ public class DoraWorkerClientServiceHandler extends BlockWorkerGrpc.BlockWorkerI
   private final ReadResponseMarshaller mReadResponseMarshaller = new ReadResponseMarshaller();
   private final DoraWorker mWorker;
 
+  private final LoadingCache<String, UfsFileStatus> mUfsFileStatusCache;
+
   /**
    * Creates a new implementation of gRPC BlockWorker interface.
-   *
    * @param workerProcess the worker process
    */
   public DoraWorkerClientServiceHandler(WorkerProcess workerProcess) {
     mWorker = workerProcess.getWorker(DoraWorker.class);
+    AlluxioConfiguration conf = Configuration.global();
+    UnderFileSystem ufs = UnderFileSystem.Factory.create(
+        conf.getString(PropertyKey.DORA_CLIENT_UFS_ROOT),
+        UnderFileSystemConfiguration.defaults(conf));
+    mUfsFileStatusCache = CacheBuilder.newBuilder()
+        .maximumSize(conf.getInt(PropertyKey.DORA_UFS_FILE_STATUS_CACHE_SIZE))
```

Comment:
nit: we can just Configuration.getInt and save one line above

Reply:
nice, done, thanks!